.env.config

```diff
@@ -14,11 +14,15 @@ JWT_SECRET='shhhh super secret code here bro'
 ROOT_URL='http://localhost:3000'
 
 # Credentials for the admin user created at app initialization
-ADMIN_EMAIL='admin@hackmit.org'
+ADMIN_EMAIL='admin@example.com'
 ADMIN_PASS='party'
 
 # Used to send verification, registration, and confirmation emails
-EMAIL_CONTACT='HackMIT Team <hackmit.reg.system@gmail.com>'
+EMAIL_ADDRESS='team@example.com'
+HACKATHON_NAME='Hackathon'
+TWITTER_HANDLE='hackathon'
+FACEBOOK_HANDLE='hackathon'
+EMAIL_CONTACT='Hackathon Team <team@example.com>'
```

Comment:
Can you add a comment to differentiate `EMAIL_ADDRESS` and `EMAIL_CONTACT`? Perhaps also some comments to separate config values that are use for display and those for the actual email configuration.